demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -177,9 +180,10 @@ const BACKGROUND_BLUR_ASSET_SPEC = (BACKGROUND_BLUR_ASSET_GROUP || BACKGROUND_BL
   revisionID: BACKGROUND_BLUR_REVISION_ID,
 }
 
-type VideoFilterName = 'Emojify' | 'CircularCut' | 'NoOp' | 'Segmentation' | 'Resize (9/16)' | 'Background Blur 10% CPU' | 'Background Blur 20% CPU' | 'Background Blur 30% CPU' | 'Background Blur 40% CPU' | 'Background Replacement' | 'None';
+type VideoFilterName = 'Premium Device - Blue Shift' | 'Premium Device - Red Shift' | 'Emojify' | 'CircularCut' | 'NoOp' | 'Segmentation' | 'Resize (9/16)' | 
+  'Background Blur 10% CPU' | 'Background Blur 20% CPU' | 'Background Blur 30% CPU' | 'Background Blur 40% CPU' | 'Background Replacement' | 'None';
 
-const VIDEO_FILTERS: VideoFilterName[] = ['Emojify', 'CircularCut', 'NoOp', 'Resize (9/16)'];
+const VIDEO_FILTERS: VideoFilterName[] = ['Premium Device - Blue Shift', 'Premium Device - Red Shift', 'CircularCut', 'NoOp', 'Resize (9/16)'];
```

Comment:
Can you use a better name such as: `SELECTED_VIDEO_FILTERS` or `DEFAULT_VIDEO_FILTERS` depends on what exactly these are for. Just `VIDEO_FILTERS` are not clear

Reply:
Changed to DEFAULT_VIDEO_FILTERS